tests/test_tools_mus.py

```diff
@@ -2,10 +2,15 @@
 from unittest import TestCase
 
 from cpmpy import *
```

Comment:
Change `import cpmpy as cp`, especially write cp.all for the “all” method that is sometimes overloaded sometimes not.